src/Wpf/Prism.DryIoc.Wpf/PrismBootstrapper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using DryIoc;
+using Prism.Ioc;
+using System;
+
+namespace Prism.DryIoc
+{
+    public abstract class PrismBootstrapper : PrismBootstrapperBase
+    {
+        /// <summary>
+        /// Create <see cref="Rules" /> to alter behavior of <see cref="IContainer" />
+        /// </summary>
+        /// <returns>An instance of <see cref="Rules" /></returns>
+        protected virtual Rules CreateContainerRules() => Rules.Default.WithAutoConcreteTypeResolution()
+                                                                       .With(Made.Of(FactoryMethod.ConstructorWithResolvableArguments))
+                                                                       .WithDefaultIfAlreadyRegistered(IfAlreadyRegistered.Replace);
```

Comment:
we're doing this in too many places... I'm wondering if we should move the default Rules to a static property off of the DryIocContainerExtension? Granted we would probably be doing some #if SomePlatform to to handle minor differences between mobile/desktop

Reply:
yeah, that should probably be a separate issue. We have the same problem for other things like region adapters, require types, etc. that are being used on both the App and Bootstrapper classes.